_index.md

```
@@ -128,7 +128,11 @@ summary: TiDB 是 {{{ .company }}} 公司自主设计、研发的开源分布式
 
 [系统变量](https://docs.pingcap.com/zh/tidb/dev/system-variables)
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)